_opensearch/query-dsl/term.md

```diff
@@ -310,6 +310,9 @@ Parameter | Behavior
 `lte` | Less than or equal to.
 `lt` | Less than.
 
+In addition to the range query parameters, you can provide date formats or relation operators such as "contains" or "within." To see the supported field types for range queries, see [Range query optional parameters]({{site.url}}{{site.baseurl}}/opensearch//supported-field-types/range/#range-query-optional-parameters).
```

Comment:
Do we have a link to these date formats? Maybe to here, https://opensearch.org/docs/latest/opensearch/supported-field-types/date/#formats?